docs/src/main/paradox/grpc.md

```diff
@@ -173,7 +230,9 @@ In this example the decision is based on tags, but the filter function can use a
 on the total quantity of the shopping cart, which requires the full state of the shopping cart and is not known from
 an individual event.
 
-Note that the purpose of the `withProducerFilter` is to toggle if all events for the entity are to be emitted or not.
+@ref:[Topic filters](#topics) can be defined in similar way, using `withTopicProducerFilter`. 
+
+Note that the purpose of `withProducerFilter` and `withTopicProducerFilter` is to toggle if all events for the entity are to be emitted or not.
```

Comment:
Maybe pull this up, or duplicate also up directly in the "Topics" section. I expect it is a bit of a different from MQTT where you'd filter for individual events matching the topic, not all events from the same source/producer of events?

Reply:
added a `@@@ note` at the top of the Filters section, that is the case for all filters
https://github.com/akka/akka-projection/pull/940/commits/1b1b88709aec7475d2001d181561ae09022a1b8b